set.mm

```diff
@@ -85886,13 +85950,31 @@ general ordinal versions of these theorems (in this case ~ oa0r ) so
       VASURVEVNVHNVFVEVMVHADWDVJVLVHWDVJFWBVLVHWDVJWBWHUSWCUTVBVCSVD $.
   $}
 
-  $( One is a natural number.  (Contributed by NM, 29-Oct-1995.) $)
+  $( The ordinal 1 is a natural number.  (Contributed by NM, 29-Oct-1995.)
+     Avoid ~ ax-un .  (Revised by BTernaryTau, 1-Dec-2024.) $)
   1onn $p |- 1o e. _om $=
+    ( vx c1o com wcel con0 cv wlim wi wal 1on 1ellim ax-gen elom mpbir2an ) BCD
+    BEDAFZGBODHZAIJPAOKLABMN $.
+  $( $j usage '1onn' avoids 'ax-un'; $)
+
+  $( Obsolete version of ~ 1onn as of 1-Dec-2024.  (Contributed by NM,
+     29-Oct-1995.)  (Proof modification is discouraged.)
+     (New usage is discouraged.) $)
+  1onnOLD $p |- 1o e. _om $=
```

Comment:
I think 1onnOLD and 2onnOLD should be kept as ALT theorems, because they are based on the theorems peano1 and peano2, which is the straightforward way to prove them.

Reply:
Done.